### ISSUE TITLE
Refinar la búsqueda de adminbirras

### ISSUE THREAD

**Issue by Nachichuri** [sysarmy/bot.ai#3]
Trajo un evento que no es adminbirras

![image](https://github.com/user-attachments/assets/ba7d743c-3857-4143-92d8-e2f48ea5eba5)

**Comment by maragunde** [comment]
Claro, la API trae todo lo que este scheduleado en el calendar de sysarmy (usa ics de google). Me lo marco. La idea seria que !birras solamente traiga adminbirras? Sirve que traiga todos los eventos o confunde?

**Comment by Nachichuri** [comment]
Capaz es un `if "birras" in title.lower()` nomas y nos aseguramos que solo traiga las birras que esten scheduleadas

**Comment by maragunde** [comment]
Fixed en release 1.2
